Bio/AlignIO/FastaIO.py

```diff
@@ -168,8 +168,10 @@ def build_hsp():
         if alphabet == single_letter_alphabet and "sq_type" in query_tags:
```

Comment:
I guess we should store the molecule type regardless of what the `alphabet` is.

Reply:
Good catch, yes.